src/rgw/driver/sfs/types.cc

```diff
@@ -93,6 +100,63 @@ Object* Object::create_commit_new_object(const rgw_obj_key& key, SFStore* store,
   return result;
 }
 
+Object* Object::try_create_with_last_version_from_database_fetch(
```

Comment:
This name makes my head hurt. Did you mean something like `try_create_with_last_version_fetch_from_database` ?

Reply:
:+1: